internal/ingress/controller/store/store.go

```diff
@@ -778,6 +778,11 @@ func (s *k8sStore) ListIngresses(filter IngressFilterFunc) []*ingress.Ingress {
 	sort.SliceStable(ingresses, func(i, j int) bool {
 		ir := ingresses[i].CreationTimestamp
 		jr := ingresses[j].CreationTimestamp
+		if ir.Equal(&jr) {
+			in := fmt.Sprintf("%v-%v", ingresses[i].Namespace, ingresses[i].Name)
```

Comment:
Please add a warning here with information about the two ingresses. This should be an exception.
Squash and then this is ready to be merged

Reply:
done